cypress/support/api/users.js

```diff
@@ -141,6 +145,54 @@ Cypress.Commands.add('createTempUser', (permissions = [], patronGroupName, userT
               } else cy.updateRolesForUserApi(userProperties.userId, [roleId]);
             });
           } else {
```

Comment:
I think it would be better to add a condition `if (Cypress.env('eureka'))` so that capability assignment would only work on Eureka. That way it would be easier to merge this branch to master later on.